flexiblepricing/templates/mail/flexible_price/body.html

```diff
@@ -1,7 +1,10 @@
 {% extends "email_base.html" %}
 
 {% block content %}
-
-{{body}}
-
+    <br>
+    <div style="text-align: left">
+        <p style="font-weight: normal; margin: 0 0 20px;">Dear {{ first_name }},</p>
+        <p style="font-weight: normal; margin: 0 0 20px;">{{ message }}</p>
+        <p style="font-weight: normal; margin: 0 0 20px;">Thank you,<br> "The {{program_name}} MITxOnline team"</p>
+    </div>
```

Comment:
Just curious if it's possible to add the styling to email_base.html instead of inline styling? I know I had issue to apply class styling in the email template in the past

Reply:
I was worried about making too many changes to the base and impacting other emails which rely on the base.  What was the issue you've had in the past, @rachellougee ?

Reply:
sorry I wasn't clear. I just meant we could give a class for this div, and add styling in the <style> section assuming no existing page would pick up this custom style. I used to have issue applying class style in the email template and only worked when adding inline style,  so I was wondering if that's the reason we add inline styling here. Anyway, I don't think it's necessary. What you have is fine